### ISSUE TITLE
 access to the hyperparameters of SOM after saving it using Joblib?

### ISSUE THREAD

**Issue by Kheireddine-otm** [JustGlowing/minisom#127]
Hello, 
can i have access to the hyperparameters of SOM after saving it using Joblib?, i mean when i load the SOM again, can i access to sigma and the learning rate of the SOM again?

**Comment by JustGlowing** [comment]
hi there, sigma and learning rate are stored in these attributes:

https://github.com/JustGlowing/minisom/blob/master/minisom.py#L159

**Comment by Kheireddine-otm** [comment]
Thanks !
